src/ui-test/camelProjectCreateTest.ts

```diff
@@ -0,0 +1,95 @@
+import { InputBox, QuickOpenBox, ActivityBar, Notification, By, TreeItem, WebDriver, VSBrowser } from "vscode-extension-tester";
+import { openCommandPrompt, removeFilePathRecursively, getIndexOfQuickPickItem, notificationExists } from "./common/commonUtils";
+import * as fs from "fs";
+import { ProjectInitializer } from "./common/projectInitializerConstants";
+import { expect } from "chai";
+let os = require('os');
+let path = require('path');
+
+const CAMEL_MISSIONS_EXPECTED = [
+    'circuit-breaker', 'configmap', 'health-check', 'rest-http', 'istio-distributed-tracing'
+];
+
+const DIR = "Fuse_Camel_TestFolder";
+const RUNTIME_VERSION = "fuse redhat750";
+
+/**
+ * @author Ondrej Dockal <odockal@redhat.com>
+ */
+export function testCreatingCamelProject() {
+
+    describe('Verify Project initializer Camel/Fuse projects creation', async function() {
+
+        let homedir: string;
+        let inputBox: QuickOpenBox;
+        let driver: WebDriver;
+
+        before(async function() {
+            this.timeout(10000);
+            homedir = os.homedir() + path.sep + DIR;
+            driver = VSBrowser.instance.driver;
+            if (! fs.existsSync(homedir)) {
+                fs.mkdirSync(homedir);
+            }
+            await openCommandPrompt();
+            const quick = await QuickOpenBox.create();
+            await quick.setText(">Extest: Add Folder");
+            await quick.confirm();
+            let confirmedPrompt = await InputBox.create();
+            await confirmedPrompt.setText(homedir);
+            await confirmedPrompt.confirm();
+        });
+
+        CAMEL_MISSIONS_EXPECTED.forEach( async function(mission) {
+            it('Test creating Camel/Fuse project: ' + mission + ", runtime & version: " + RUNTIME_VERSION, async function () {
+                this.timeout(20000); 
```

Comment:
Can you try to use a conditional wait?
Have you spotted something missing to use ones?